wagtail/images/management/commands/wagtail_update_image_renditions.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
+from wagtail.images.models import Rendition, Image
```

Comment:
We will need to likely use the `get_image_model`  so this code can support custom image models

https://docs.wagtail.org/en/stable/advanced_topics/images/custom_image_model.html#module-wagtail.images